src/kOS.Safe/Encapsulation/Structure.cs

```diff
@@ -157,5 +158,59 @@ public override string ToString()
         {
             return "Structure ";
         }
+
+        public static StringValue operator +(Structure val1, Structure val2)
+        {
+            return new StringValue(val1.ToString() + val2.ToString());
+        }
+
+        public static object FromPrimitive(object value)
```

Comment:
I don't know that I'm convinced it's the perfect class for it, but I've never been huge on "utility" classes that exist simply to provide miscellaneous helper methods.   I didn't want three separate methods on `StringValue`, `ScalarValue`, and `BooleanValue` because that would just mean chaining together a bunch of convert attempts.   My logic for putting it into the `Structure` class was that it was the first common inherited class, and it is used primarily to ensure an object is a `Structure` so that it has suffixes.